bigquery/snippets/snippet.go

```diff
@@ -16,6 +16,113 @@ import (
 	"google.golang.org/api/iterator"
 )
 
+// Use a common block to inline comments related to importing the library
```

Comment:
Wheee!

Reply:
I can see this getting pretty annoying to match up start and end tags, but I get why we might want to put this in a central place.

Reply:
I use the same pattern for the shared runAndRead function that inserts a query job and reads result rows.  If it grows too hairy we can consider doing something clever with templates in the docsite markup.